packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -22,7 +22,7 @@ import { store as editPostStore } from '../../store';
 import { unlock } from '../../lock-unlock';
 import { usePaddingAppender } from './use-padding-appender';
 
-const { EditorCanvas } = unlock( editorPrivateApis );
+const { VisualEditor: VisualEditorRoot } = unlock( editorPrivateApis );
```

Comment:
This PR includes a rename to this component as it's clearer this way. The canvas can hold both code and visual editors but what we used to call "canvas" represents just the visual editor actually.